Description: Fix testcase equatenosuchstn
 The expected output hardcodes version "1.2.5", so fails in 1.2.6.

 Patch backported from upstream git.
Author: Olly Betts <olly@survex.com>

---
Last-Update: 2012-02-28

--- a/tests/cavern.tst	2012-02-28 12:38:37.000000000 +1300
+++ b/tests/cavern.tst	2012-02-24 14:30:21.000000000 +1300
@@ -59,7 +59,7 @@
 
 LC_ALL=C
 export LC_ALL
-unset SURVEXLANG
+SURVEXLANG=
 export SURVEXLANG
 
 for file in $TESTS ; do
@@ -268,9 +268,9 @@
     if test -f "$file.out" ; then
       # Check output is as expected.
       if test -n "$VERBOSE" ; then
-	diff tmp.out "$file.out" || exit 1
+	sed '1,/^$/d' tmp.out|diff - "$file.out" || exit 1
       else
-	diff tmp.out "$file.out" > /dev/null || exit 1
+	sed '1,/^$/d' tmp.out|diff - "$file.out" > /dev/null || exit 1
       fi
     fi
     rm -f tmp.*
--- a/tests/equatenosuchstn.out	2012-02-28 12:38:37.000000000 +1300
+++ b/tests/equatenosuchstn.out	2012-02-24 13:38:00.000000000 +1300
@@ -1,6 +1,3 @@
-Survex 1.2.5
-Copyright (C) 1990-2012 Olly Betts
-
 Removing trailing traverses...
 
 Concatenating traverses between nodes...
